calour/_testing.py

```diff
@@ -17,6 +18,9 @@
 
 class Tests(TestCase):
     def setUp(self):
+        # disable logging
+        logging.disable(logging.CRITICAL)
```

Comment:
why disable logging.critical ? what does this line do?

Reply:
to suppress the log msg in all the tests. not critical, just make the screen cleaner when running tests.

Reply:
i like to have the log messages since it helps find out the bug when tests fail - but whatever you prefer.

Reply:
the unit tests are already very informative in printing error msg...